sct.py

```diff
@@ -646,15 +646,16 @@ def list_resources_on_azure():
               type=str,
               help="List images by version. Use '-v all' for all versions. "
                    "OSS format: <4.3> Enterprise format: <enterprise-2021.1>. Mutually exclusive with --branch.")
-@click.option('-r', '--region',
+@click.option('-r', '--region', "regions",
               type=CloudRegion(),
               help="Cloud region to query images in. Defaults to eu-west-1",
-              default="eu-west-1")
+              default=["eu-west-1"],
```

Comment:
nitpick:
this default is a bit problematic, since it would fail any other backend but AWS.
we probably need a default per backend in code

and that probably why we had default in the code to begin with

Reply:
Changed, to use default by the region, reuse the list from NemesisJobGenerator